zerver/tests/test_events.py

```diff
@@ -426,6 +426,11 @@ def normalize(state: Dict[str, Any]) -> None:
                 state["realm_bots"] = {u["email"]: u for u in state["realm_bots"]}
             # Since time is different for every call, just fix the value
             state["server_timestamp"] = 0
+            if "presence_last_update_id" in state:
+                # We don't adjust presence_last_update_id via apply_events,
+                # since events don't carry the relevant information.
+                # Fix the value just like server_timestamp.
+                state["presence_last_update_id"] = 0
```

Comment:
We should spell out in the comment that we pick 0, which a correct client should consider a noop, since `max(0, previous_presence_last_update_id) = previous_presence_last_update_id`

Reply:
Hmm I'm not sure I like adding such a comment. We don't set it to `0` because `max(0, previous_presence_last_update_id) = previous_presence_last_update_id`. It's just like `server_timestamp`, we set it to `0` because this is a "this value doesn't matter for the purposes of these tests". I don't think this should be interpreted under the frame of "what would this value mean to a client"? (`server_timestamp=0` also wouldn't be a meaningful value to a client; if anything, it's be just wrong).

And actually a client should be confused by seeing `presence_last_update_id=0` in almost any scenario - e.g. if there's no new presence data in the server response, this value will be equal to the `presence_last_update_id` it sent, not to `0`.

So as a reader, I'd find this kind of comment confusing due to it suggesting me to interpret a value that's not  actually meant to mean anything?